device-types/Ciena/8112-dc.yaml

```diff
@@ -0,0 +1,115 @@
+---
+manufacturer: Ciena
+model: 8112 DC
+slug: ciena-8112-dc
+part_number: 170-8112-900
+u_height: 1
+is_full_depth: false
+weight: 7
+weight_unit: kg
+subdevice_role: parent
```

Comment:
Generally you do not need `subdevice_role` unless the device either slots into another (in which case it would be a child), or has `device-bays` to house child devices. If this device does not need this, please remove it